name-computer/map.jinja

```diff
@@ -12,6 +12,9 @@
     salt.pillar.get('name-computer:lookup:computername') | default('', true)
 ) %}
 
+{%- set dns_domain = salt.pillar.get('join-domain:lookup:dns_name') | default('', true) %}
```

Comment:
Can't rely on a different formula's namespace. Need to add a new pillar/grain variable lookup under the name-computer namespace

Reply:
Euw... Alright

Reply:
Addressed in 0efde0ee